regression/regression_utils.py

```diff
@@ -31,6 +32,38 @@
 """
 
 
+class RetryTestsMetaclass(type):
+
+    NUM_RETRIES = 2
+    FLAKY_ERROR_CLASSES = (AssertionError,)
```

Comment:
RE: "Derived classes." Can we cover it if we need it?

RE: "Flakiness of `AssertionError`." It depends on the test. Rather than dig up old flaky instances, I figured we'd start with something we know could be problematic, like `self.assertEqual(len(thing), 6)`, and then expand as needed.